ccmbioinfo/lfs
# CARDS-1341: Create a PROMs run mode

This PR adds the `proms` runMode that, when enabled (eg. `-Dsling.run.modes=proms`), assigns the name _Cards for PROMs_ to the web application and makes the _PROMs_ Questionnaires available. Additionally, this PR causes the _cancer cells_ background to only be displayed when in the `lfs` runMode. When in the `kids` runMode, the _Cards4Kids_ background is displayed. For all other runModes, a plain black background is displayed.

To test:

1. Build this branch (`mvn clean install`)
2. Start CARDS without specifying any runModes. The web application should be named _Clinical ARchive for Data Science_ and the sidebar background should be plain black.
3. Stop CARDS and `rm -rf sling`
4. Start CARDS with the `lfs` runMode specified. The web application should be named _LFS Data Core_ and the sidebar background should be the cancer cells image.
5. Stop CARDS and `rm -rf sling`
6. Start CARDS with the `kids` runMode specified. The web application should be named _WilliamsDB_ and the sidebar background should be an image of a heart containing ones and zeros.
7. Stop CARDS and `rm -rf sling`
8. Start CARDS with the `cardiac_rehab` runMode specified. The web application should be named _Cards 4 CaRe_ and the sidebar background should be plain black.
9. Stop CARDS and `rm -rf sling`
10. Start CARDS with the `proms` runMode specified. The web application should be named _Cards for PROMs_ and the sidebar background should be plain black. Additionally, the PROMs Questionnaires should be available and usable to create Forms. Please note that _Smoking cessation_ Forms will always be marked as `INCOMPLETE` - this is a known bug (CARDS-742) but is out-of-scope of this PR.
